src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/CaseDetailsSearchOperation.java

```diff
@@ -2,12 +2,9 @@
 
 import uk.gov.hmcts.ccd.domain.model.search.CaseDetailsSearchResult;
 
-import java.io.IOException;
-import java.util.List;
-
 
 public interface CaseDetailsSearchOperation {
 
-    CaseDetailsSearchResult execute(List<String> caseTypeIds, String query) throws IOException;
+    CaseDetailsSearchResult execute(String caseTypeId, String jsonQuery);
```

Comment:
shall we rename `jsonQuery` to `searchRequest`? because from the controller we pass down the whole request not only the query

Reply:
Passing the new request object everywhere CaseSearchRequest